packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue

```diff
@@ -237,33 +242,33 @@ export default {
          * Takes the locale and lazyloads the correct date locale from the date-fns library
          * @param locale
          * @returns {
-         *  Promise<module:date-fns/locale/da> |
-         *  Promise<module:date-fns/locale/en-AU> |
-         *  Promise<module:date-fns/locale/en-GB> |
-         *  Promise<module:date-fns/locale/en-NZ> |
-         *  Promise<module:date-fns/locale/es> |
-         *  Promise<module:date-fns/locale/it> |
-         *  Promise<module:date-fns/locale/nb>
+         *  da |
+         *  enAU |
+         *  enGB |
+         *  enNZ |
+         *  es |
+         *  it |
+         *  nb
          * }
          */
         getDateFnsLocale (locale) {
             switch (locale) {
                 case 'da-DK':
-                    return import(/* webpackChunkName: "date-fns-locale-da" */ 'date-fns/locale/da');
+                    return da;
                 case 'en-AU':
-                    return import(/* webpackChunkName: "date-fns-locale-en-AU" */ 'date-fns/locale/en-AU');
+                    return enAU;
                 case 'en-GB':
                 case 'en-IE':
                 default:
```

Comment:
I've not seen the default case in the middle before but I'm assuming it works.
I think it would probably be a little more standard to have `default: return enGB` at the end